packages/core/collections/tags.js

```diff
@@ -72,15 +72,15 @@ export default class Tags extends Collection {
     });
   }
 
-  async rename(tagId, newName) {
+  async rename(tagId, newName, isColorTag) {
     let tag = this.tag(tagId);
     if (!tag) {
       console.error(`No tag found. Tag id:`, tagId);
       return;
     }
 
-    newName = this.sanitize(newName);
-    if (!newName) throw new Error("Tag title cannot be empty.");
+    if (!isColorTag) newName = this.sanitize(newName);
```

Comment:
There is no need to do any sanitization in any case during renaming.

Reply:
Ok